docs/_posts/examples/3400-02-03-heatmap-layer.html

```diff
@@ -0,0 +1,122 @@
+---
+layout: example
+category: example
+title: Create a heatmap layer
+description: Visualize earthquake frequency by location using a heatmap layer.
+tags:
+  - layers
+---
+
+<div id='map'></div>
+
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/dark-v9',
+    center: [-120, 50],
+    zoom: 2
+});
+
+map.on('load', function() {
+    //Add a geojson point source.  
+    //Heatmap layers also work with a vector tile source.
+    map.addSource('earthquakes', {
+        "type": "geojson",
+        "data": "{{site.baseurl}}/assets/earthquakes.geojson"
+    });
+
+    map.addLayer({
+        "id": "earthquakes-heat",
+        "type": "heatmap",
+        "source": "earthquakes",
+        "maxzoom": 9,
+        "paint": {
+            //Increase the heatmap weight based on frequency and property magnitude
+            "heatmap-weight": {
+                "property": "mag",
+                "type": "exponential",
+                "stops": [
+                    [0, 0],
+                    [6, 1]
+                ]
+            },
+            //Increase the heatmap color weight weight by zoom level
+            //heatmap-ntensity is a multiplier on top of heatmap-weight
+            "heatmap-intensity": {
+                "stops": [
+                    [0, 1],
+                    [9, 3]
+                ]
+            },
+            //Color ramp for heatmap.  Domain is 0 (low) to 1 (high)
+            "heatmap-color": {
+                "stops": [
+                    [0, "#2166ac"],
```

Comment:
Let's add a 0-alpha stop a 0. E.g. we can try `#2166ac00`

Reply:
@mourner updated the ramp to start with transparency zero.  

![](https://cl.ly/0x2a0H3K0z1g/download/Screen%20recording%202017-10-08%20at%2006.58.41%20PM.gif)